ui/src/main/kotlin/edu/artic/ui/BaseFragment.kt

```diff
@@ -123,4 +129,9 @@ abstract class BaseFragment : Fragment() {
     }
 
 
-}
\ No newline at end of file
+}
+
+/**
+ * Marker interface to mark those fragments which shouldn't modify the parent activity theme.
+ */
+interface AloofFragment
```

Comment:
Give it a better name. or better yet, just use an open `val` that they can override to control this behavior.

Reply:
Oh, good idea. `open val mayOverrideStatusBarColor` or something like that?

Reply:
yeah to follow convention in other parts of it

Reply:
Replaced the marker interface by parent class property.